Build failure of ut_time_stubs with BUILDTYPE=release

**Describe the bug**
Matrix build fails with: BUILDTYPE=release OMIT_DEPRECATED=false
See also https://github.com/nasa/cFS/pull/40/ (enhanced CI)

**To Reproduce**
Steps to reproduce the behavior:
1. See CI process/results

**Expected behavior**
Successful build

**Error text**
```
home/travis/build/nasa/cFS/cfe/fsw/cfe-core/ut-stubs/ut_time_stubs.c: In function ‘CFE_TIME_Print’:
/home/travis/build/nasa/cFS/cfe/fsw/cfe-core/ut-stubs/ut_time_stubs.c:106:25: error: ‘ -’ directive output may be truncated writing 2 bytes into a region of size between 0 and 18 [-Werror=format-truncation=]
              "UT %lu.%lu -",
                         ^~
In file included from /usr/include/stdio.h:862:0,
                 from /home/travis/build/nasa/cFS/cfe/fsw/cfe-core/ut-stubs/ut_time_stubs.c:35:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:64:10: note: ‘__builtin___snprintf_chk’ output between 9 and 27 bytes into a destination of size 24
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        __bos (__s), __fmt, __va_arg_pack ());
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```

**System observed on:**
 - CI system, integration candidate bundle

**Additional context**
N/A

**Reporter Info**
Jacob Hageman - NASA/GSFC

--- a/fsw/cfe-core/ut-stubs/ut_time_stubs.c
+++ b/fsw/cfe-core/ut-stubs/ut_time_stubs.c
@@ -102,11 +102,10 @@ void CFE_TIME_TaskMain(void)
 void CFE_TIME_Print(char *PrintBuffer, CFE_TIME_SysTime_t TimeToPrint)
 {
     snprintf(PrintBuffer,
-            CFE_TIME_PRINTED_STRING_SIZE,
-             "UT %lu.%lu -",
-             (unsigned long)TimeToPrint.Seconds,
-             (unsigned long)TimeToPrint.Subseconds);
-
+             CFE_TIME_PRINTED_STRING_SIZE,
+             "UT %u.%u -",
+             (unsigned int)TimeToPrint.Seconds,
+             (unsigned int)TimeToPrint.Subseconds);
 
     UT_Stub_RegisterContext(UT_KEY(CFE_TIME_Print), PrintBuffer);
     UT_Stub_RegisterContext(UT_KEY(CFE_TIME_Print), &TimeToPrint);

